handbook/product/user_research/user_research_observer.md

```diff
@@ -0,0 +1,39 @@
+# So you’re about to help us with user testing
```

Comment:
```suggestion
# Observing user research
```

It's more descriptive than 'help us'. Helping indicates a bigger time commitment imo